File: year2020/month12/day1204/ms_1204.cpp
#include <iostream>
#include <string>
#include <algorithm>

using namespace std;

string solution(string s);

int main() {
	string str;

	getline(cin, str);

	if (1 <= str.length()) {
		cout << solution(str) << endl;
	}

	return 0;
}

string solution(string s) {
	string answer = "";

	sort(s.begin(), s.end());
	reverse(s.begin(), s.end());

	answer = s;

	return answer;
}

File: year2020/month12/day1204/ms_1204.cs
using System;

namespace algorithm_115
{
    class ms_1204
    {
        static string solution(string s) 
        {
            string answer = "";
            char[] temp = new char[s.Length];

            for(int i = 0; i < s.Length; i++)
            {
                temp[i] = s[i];
            }

            Array.Sort(temp);
            Array.Reverse(temp);

            for(int i = 0; i < s.Length; i++) 
            {
                answer += temp[i];
            }

            return answer;
        }

        static void Main(string[] args)
        {
            string str = Console.ReadLine();

            if(1 <= str.Length)
            {
                Console.WriteLine(solution(str));
            }
        }
    }
}
